### PR TITLE
Fixes #12564: It's not possible to search Nodes by their node lifecycle

### PR DESCRIPTION
**Pull request by fanf**
https://www.rudder-project.org/redmine/issues/12564

### CODE REVIEW

**Comment by fanf** [comment]
PR rebased

File: rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js [hunk 1]
```diff
@@ -486,7 +486,7 @@ function createRuleTable(gridId, data, checkboxColumn, actionsColumn, compliance
       }
     , "columnDefs": [{
           "targets": [ 0 ]
-        , "visible": false
+        , "visible": true
```
Comment (VinceMacBuche) [inline rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js]: Do you really want to change that line ?

File: rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js [hunk 2]
```diff
@@ -331,7 +331,7 @@ function createRuleTable(gridId, data, checkboxColumn, actionsColumn, compliance
   };
   var tags = {
       "mDataProp"     : "tags"
-    , "sClass"        : "never"
+//    , "sClass"        : "never"
```
Comment (VinceMacBuche) [inline rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js]: Could be removed

File: rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js (hunk 1 above)
Comment (fanf) [reply rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js]: nop :)

**Comment by fanf** [comment]
PR rebased

**Pull request merged**